apps/settings/elements/screen_lock.html

```diff
@@ -18,22 +18,20 @@ <h1 data-l10n-id="screenLock-header"></h1>
           </gaia-switch>
         </li>
         <li class="passcode-enabled" id="passcode-timeout-row">
-          <p data-l10n-id="require-passcode">Require passcode</p>
+          <p data-l10n-id="require-passcode"></p>
           <span class="button icon icon-dialog">
             <select name="lockscreen.passcode-lock.timeout">
-              <option value="0" data-l10n-id="immediately" selected>Immediately</option>
-              <option value="60" data-l10n-id="after-one-minute">After 1 minute</option>
-              <option value="300" data-l10n-id="after-five-minutes">After 5 minutes</option>
-              <option value="900" data-l10n-id="after-fifteen-minutes">After 15 minutes</option>
-              <option value="1800" data-l10n-id="after-thirty-minutes">After 30 minutes</option>
-              <option value="3600" data-l10n-id="after-one-hour">After 1 hour</option>
+              <option value="0" data-l10n-id="immediately" selected></option>
+              <option value="60" data-l10n-id="after-one-minute"></option>
+              <option value="300" data-l10n-id="after-five-minutes"></option>
+              <option value="900" data-l10n-id="after-fifteen-minutes"></option>
+              <option value="1800" data-l10n-id="after-thirty-minutes"></option>
+              <option value="3600" data-l10n-id="after-one-hour"></option>
             </select>
           </span>
         </li>
         <li class="passcode-enabled">
-          <button class="passcode-edit" data-l10n-id="change-passcode">
-            Change Passcode
-          </button>
+          <button class="passcode-edit" data-l10n-id="change-passcode"></button>
```

Comment:
If we are going to remove all strings in HTML, I suggest we file a bug and remove all of them in that bug, instead of partially doing in a unrelated bug.